syft/pkg/cataloger/dotnet/parse_dotnet_portable_executable_test.go

```diff
@@ -48,17 +49,18 @@ func TestParseDotnetPortableExecutable(t *testing.T) {
 			name: "dotnet package with malformed field and extended version",
 			versionResources: map[string]string{
 				"CompanyName":      "Microsoft Corporation",
-				"FileDescription":  "äbFile\xa0\xa1Versi on",
+				"FileDescription":  "äb\x01File\xa0\xa1Versi on",
```

Comment:
I'm not entirely sure, but these values seem like they were intentional, maybe add a second test with different ones?